Microsoft.Toolkit.Uwp.UI/Extensions/ListViewBaseExtensions.cs

```diff
@@ -0,0 +1,106 @@
+﻿using System.Windows.Input;
+using Windows.UI.Xaml;
+using Windows.UI.Xaml.Controls;
+
+namespace Microsoft.Toolkit.Uwp.UI
+{
+    /// <summary>
+    /// Provides attached dependency properties for the <see cref="ListViewBase"/> content element that allows
+    /// it to invoke a <see cref="ICommand"/> when clicked
+    /// </summary>
+    public static class ListViewBaseExtensions
+    {
+        /// <summary>
+        /// Attached <see cref="DependencyProperty"/> for binding an <see cref="ICommand"/> instance to a <see cref="ListViewBase"/>
+        /// </summary>
+        public static readonly DependencyProperty CommandProperty = DependencyProperty.RegisterAttached(
+            "Command",
```

Comment:
I wonder whether this should be more descriptive. Command on it's own normally means when the item (eg, button) is clicked, so my thinking would be the Command is triggered when the ListViewBase is clicked, not the individual ListItem is clicked. Maybe I'm being too pedantic, I don't know :)